scripts/nolints.txt

```diff
@@ -3242,6 +3241,10 @@ trunc.map
 trunc.rec
 trunc.rec_on
 trunc.rec_on_subsingleton
+turing.TM0.stmt.inhabited
```

Comment:
This is clearly not the ideal solution, however virtually all of the definitions in `turing_machine.lean` have duplicate and unused arguments (even the inductives have unused parameters).  These linting errors here are for automatically generated instances, which should  be fine as soon as the unused parameters in the inductives are removed.  A clean-up of the file will have to wait for another PR.